### PR TITLE
CHK-41 md5

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: checkmd5.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        return
+    else:
+        graph = r.json()['@graph']
+
+    excluded_statuses = ['uploading', 'upload failed', 'content error']
+    md5dictionary = defaultdict(set)
+    for f in graph:
+        if f.get('status') not in excluded_statuses:
+            md5 = f.get('md5sum')
+            md5dictionary[md5].add(f.get('uuid'))
```
Comment (paul-sud) [inline checkmd5.py]: These two lines can potentially add an entry to `md5dictionary` with a key of `None` if the file does not have an md5sum, is that OK?

File: checkmd5.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            channel="#bot-reporting",
+            text=finishing_run,
+            as_user=True,
+        )
+
+
+def main():
+    import argparse
+    parser = argparse.ArgumentParser(
+        description="Update experiments status",
```
Comment (paul-sud) [inline checkmd5.py]: Description seems wrong.

**Review by paul-sud — CHANGES_REQUESTED** [review]
Just a couple minor things, otherwise looks good.

File: checkmd5.py (hunk 1 above)
Comment (gabdank) [reply checkmd5.py]: you are right, it is not OK. Fixed.

File: checkmd5.py (hunk 2 above)
Comment (gabdank) [reply checkmd5.py]: fixed

**Comment by gabdank** [comment]
@p-sud could you please review PR, I have addressed your comments

**Review by paul-sud — APPROVED** [review]

**Pull request merged**